lib/Wikia/src/CircuitBreaker/CircuitBreakerStorage.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Wikia\CircuitBreaker;
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\Helios\Exception;
+
+class CircuitBreakerOpen extends ClientException {
+	/** @var string */
+	private $serviceName;
+
+	/**
+	 * CircuitBreakerOpen constructor.
+	 * @param string $serviceName
+	 * @param int $code
+	 * @param Exception|null $previous
+	 * @param null $data
+	 */
+	public function __construct(string $serviceName, int $code = 0, Exception $previous = null, $data = null) {
+		$this->serviceName = $serviceName;
+
+		parent::__construct("circuit breaker open for service $serviceName", $code, $previous, $data);
+	}
+
+	protected function logMe() {
+		$this->warning( 'circuit breaker open', [ 'service' => $this->serviceName ] );
+	}
+}
+
+interface CircuitBreakerStorage {
+	/**
+	 * @param string $name
+	 * @return bool
+	 */
+	public function OperationAllowed( string $name );
```

Comment:
Minor: All methods should be `lowerCamelCase`

Reply:
fix: 576c54a3c71112df5c515593963739d324e2b00b